parts/k8s/windowsazurecnifunc.ps1

```diff
@@ -255,3 +255,38 @@ function GenerateAzureStackCNIConfig
     Set-ItemProperty -Path $azureCNIConfigFile -Name IsReadOnly -Value $true
 }
 
+function New-ExternalHnsNetwork {
+    Write-Log "Creating new HNS network `"ext`""
+    $externalNetwork = "ext"
+    $na = @(Get-NetAdapter -Physical)
+
+    if ($na.Count -eq 0) {
+        throw "Failed to find any physical network adapters"
+    }
+
+    # If there is more than one adapter, use the first adapter.
+    $managementIP = (Get-NetIPAddress -ifIndex $na[0].ifIndex -AddressFamily IPv4).IPAddress

```

Comment:
@pradipd - should this be smarter? @dineshgovindasamy said there may be cases where a NIC doesn't have an IP and we should choose another?

Reply:
This is re: https://github.com/Azure/aks-engine/issues/2281

Reply:
Yes. Actually, the (Get-NetAdapter -Physical) on L261 needs to be different for acceleratedNetworkingEnabledWindows.
I just haven't had a chance to figure out what it should be.

Reply:
Yes. Actually, the (Get-NetAdapter -Physical) on L261 needs to be different for acceleratedNetworkingEnabledWindows.
I just haven't had a chance to figure out what it should be.

Reply:
@pradipd ok - we'll consider that for a separate PR